x/mint/params.go

```diff
@@ -14,6 +15,20 @@ type Params struct {
 	InflationMax        sdk.Dec `json:"inflation_max"`         // maximum inflation rate
 	InflationMin        sdk.Dec `json:"inflation_min"`         // minimum inflation rate
 	GoalBonded          sdk.Dec `json:"goal_bonded"`           // goal of percent bonded atoms
+	BlocksPerYear       int64   `json:"blocks_per_year"`       // expected blocks per year
+}
+
+func NewParams(mintDenom string, inflationRateChange, inflationMax,
+	inflationMin, goalBonded sdk.Dec, blocksPerYear int64) Params {
+
+	return Params{
+		MintDenom:           mintDenom,
+		InflationRateChange: inflationRateChange,
+		InflationMax:        inflationMax,
+		InflationMin:        inflationMin,
+		GoalBonded:          goalBonded,
+		BlocksPerYear:       blocksPerYear,
```

Comment:
How are we setting this BlocksPerYear number?

Reply:
It's just estimated - as 5 sec/block for now (in params store so governance can update if that get's to slow) - REF #2846 for making this a on-chain calculation (post-launch probs)